### DISCUSSION TITLE
Idea: Allow switching between LCA and operational emission factors

### DISCUSSION THREAD

**Discussion by madsnedergaard** [electricitymaps/electricitymaps-contrib#6787] [Ideas]
**Is your feature request related to a problem? Please describe.**

There are arguments for and against using Life Cycle Assessment (LCA) numbers and operational/direct emission factors. We should consider if this is a feature that it would make sense to implement.

**Describe the solution you'd like**
A toggle that switches between operational and direct emission factors. It should be stated very clearly on every single graph and data-point what "mode" is used to avoid people sharing screenshots that can be misread/misunderstood.

**Inputs needed**

- What are the use cases where this view would make sense?
- Does the added benefits out-weight the added complexity for non-energy experts using the app?
- How much technical work is required to make this happen?

**Additional context**

- See our documentation here for an explanation of the different approaches: https://github.com/electricitymaps/electricitymaps-contrib/wiki/Default-emission-factors
- Past discussions (but also a lot of off-topic comments): #960

**Comment by VIKTORVAV99** [comment]
Obviously some shaping required but an idea to reduce the confusion between which is used would be to include both in the tooltips like here:
<img width="367" alt="image" src="https://github.com/electricitymaps/electricitymaps-contrib/assets/30777521/d6ead790-05e8-4507-8aec-dc9b9eda6329">
Where the selected mode is highlighted but the non selected mode is still clear and visible in some way.
